modin/core/storage_formats/pandas/parsers.py

```diff
@@ -663,43 +663,66 @@ class ParquetFileToRead(NamedTuple):
 
 @doc(_doc_pandas_parser_class, data_type="PARQUET data")
 class PandasParquetParser(PandasParser):
+    @staticmethod
+    def _read_row_group_chunk(
+        f, row_group_start, row_group_end, columns, engine
+    ):  # noqa: GL08
+        if engine == "pyarrow":
+            from pyarrow.parquet import ParquetFile
+
+            return (
+                ParquetFile(f)
+                .read_row_groups(
+                    list(
+                        range(
+                            row_group_start,
+                            row_group_end,
+                        )
+                    ),
```

Comment:
```suggestion
                        range(
                            row_group_start,
                            row_group_end,
                        )
```
can we omit `list()` call here? I think I asked this once already, but let's repeat the question

Reply:
Yeah we can. Looks like it works without `list()`.